### DISCUSSION TITLE
Can I traslate rxjs.dev to Korean?

### DISCUSSION THREAD

**Discussion by gemst0ne** [ReactiveX/rxjs#6965] [Q&A] [answered]
Hi, I'm WonSeok from South Korea,

Can I translate rxjs.dev pages into Korean and distribute them in Gitbook?

I'll make sure to mark the source!

**Comment by benlesh** [comment, marked as answer]
Core Team: Unofficial translations are welcome. We just can't bring it into our repository or stamp it as "official" because there's no way for us to verify the content.
